packages/database/src/analytics/MaterializedViewLogDatabaseModel.js

```diff
@@ -7,11 +7,12 @@ import { DatabaseModel } from '../DatabaseModel';
 
 export class MaterializedViewLogDatabaseModel extends DatabaseModel {
   async fetchSchema() {
-    if (!this.schemaPromise) {
-      const realSchemaPromise = await super.fetchSchema();
-      delete realSchemaPromise.m_row$; // Remove the m_row$ field from the schema notation
-      this.schemaPromise = realSchemaPromise;
+    this.schemaPromise = await super.fetchSchema();
+
+    if ('m_row$' in this.schemaPromise) {
+      delete this.schemaPromise.m_row$; // Remove the m_row$ field from the schema notation
     }
+
```

Comment:
Looks like this bug snuck back in a while ago (including the `m_row$` field in the schema of these models). Honestly this little manoeuvre has always been a bit hacky, and I wouldn't necessarily say it's much better now, but it's fixed again